common/production_methods/11_private_infrastructure.txt

```diff
@@ -22,7 +22,7 @@
 	}
 	country_modifiers = {
 		workforce_scaled = {
-			market_land_trade_capacity_add = 1
+			market_land_trade_capacity_add = 0.1
```

Comment:
On further thought, you can do it times 3. That should make it more impactful, and we need to decide what we want the base to be and the average infrastructure bonus.

Reply:
what do you mean by the average infrastructure bonus? I assume base refers to the base amount of land trade capacity

Reply:
I made a modifier which updates every month.